WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelPackageList: View {
+    @EnvironmentObject private var viewModel: ShippingLabelPackageDetailsViewModel
+    @Environment(\.presentationMode) var presentation
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
+
+                /// Custom Packages
+                ///
+                if viewModel.customPackages.count > 0 {
+                    ListHeaderView(text: Localization.customPackageHeader.uppercased(), alignment: .left)
+                        .background(Color(.listBackground))
+                }
+                ForEach(viewModel.customPackages, id: \.title) { package in
+                    let selected = package == viewModel.selectedCustomPackage
+                    SelectableItemRow(title: package.title, subtitle: package.dimensions + " \(viewModel.dimensionUnit)", selected: selected).onTapGesture {
+                        viewModel.didSelectPackage(package.title)
+                    }
+                    Divider().padding(.leading, Constants.dividerPadding)
+                }
+
+                /// Predefined Packages
+                ///
+                ForEach(viewModel.predefinedOptions, id: \.title) { option in
```

Comment:
Same comment about the `id` as above!

Reply:
Like above, 663444b4f4d3b2693b0657a55807d4de511d42a4

Reply:
Also here, the title of the option is unique 👍 321ade2aad9a1b78108445fc1829b1a3d0f2a5ba